integration_setup.sh

```diff
@@ -128,6 +128,40 @@ isOpenProjectFileStorageConfigOk() {
   fi
 }
 
+logUnhandledError() {
+	log_error "Unhandled error while creating the file storage '${OPENPROJECT_STORAGE_NAME}'"
+	log_error "OpenProject returned the following error: '${error_message}'"
+	log_info "You could try deleting the file storage '${OPENPROJECT_STORAGE_NAME}' in OpenProject and run the script again."
+}
+
+checkForOpenProjectOrNextcloudIntegrationConfiguration() {
```

Comment:
```suggestion
checkForOpenProjectAndNextcloudIntegrationConfiguration() {
```

Or even better split into two functions, one for OP, one for NC. Every function just does one thing

Reply:
Ah sure! I will may be first do that on release/2.7 and then do it here again.